packages/fuels-core/src/utils.rs

```diff
@@ -13,3 +21,65 @@ pub fn first_four_bytes_of_sha256_hash(string: &str) -> ByteArray {
     output[4..].copy_from_slice(&result[..4]);
     output
 }
+
+pub fn try_from_bytes<T>(bytes: &[u8]) -> Result<T>
```

Comment:
This is also intimately related to stuff over in the new `codec` mod, maybe we should move it there and leave this file empty?